include/mbedtls/ssl.h

```diff
@@ -571,6 +571,8 @@
 #define MBEDTLS_TLS_EXT_ENCRYPT_THEN_MAC            22 /* 0x16 */
 #define MBEDTLS_TLS_EXT_EXTENDED_MASTER_SECRET  0x0017 /* 23 */
 
+#define MBEDTLS_TLS_EXT_RECORD_SIZE_LIMIT           28 /* RFC 8449 (implemented for TLS 1.3 only) */
```

Comment:
```suggestion
#define MBEDTLS_TLS_EXT_RECORD_SIZE_LIMIT           28 /* RFC 8449 TLS 1.3 */
```

Reply:
I disagree with this comment - this can also apply to TLS 1.2, but it was Janos' choice that only 1.3 is now implemented (in [the issue](https://github.com/Mbed-TLS/mbedtls/issues/7007)).

Reply:
I agree with @AndrzejKurek. Thus, keeping it as it is for now.